src/flatbuffers/metric_common.fbs

```diff
@@ -0,0 +1,57 @@
+namespace circonus;
+
+table IntValue {
+      value:int (id: 0);
+}
+
+table UintValue {
+      value:uint (id: 0);
+}
+
+table LongValue {
+      value:long (id: 0);
+}
+
+table UlongValue {
+      value:ulong (id: 0);
+}
+
+table DoubleValue {
+      value:double (id: 0);
+}
+
+table StringValue {
+      value:string (id: 0);
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentNumericValue {
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentStringValue {
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentHistogramValue {
+}
+
+table HistogramBucket {
+       val: byte;
+       exp: byte;
+       count: ulong;
+}
+
+table Histogram {
+       buckets: [HistogramBucket] (id: 0);
+}
+
+union MetricValueUnion { IntValue, UintValue, LongValue, UlongValue, DoubleValue, StringValue, Histogram, AbsentNumericValue, AbsentStringValue, AbsentHistogramValue }
+
+table MetricValue {
+      name: string (id: 0);
+      timestamp: ulong (id: 1);
```

Comment:
No, name here means `metric_name`, the `table Metric` does not have a `metric_name` field.  I repeated `timestamp` here on purpose as requested by @postwait due to http traps having incoming timestamps that differ from the check.